src/D2L.Security.OAuth2/Validation/AccessTokens/AccessTokenValidatorFactory.cs

```diff
@@ -52,5 +52,27 @@ Uri authEndpoint
 			return result;
 		}
 
+		/// <summary>
+		/// Creates an <see cref="IAccessTokenValidator"/> for Standard OAuth2 instance backed by a remote token signer.
+		/// </summary>
+		/// <param name="httpClient"><see cref="HttpClient"/> instance with which requests will be made. The lifecycle of the <see cref="HttpClient"/> is not managed. It will not be disposed by the validator.</param>
+		/// <param name="authEndpoint">The URI of the remote JWKS file</param>
+		/// <returns></returns>
+		public static IAccessTokenValidator CreateRemoteStandardValidator(

```

Comment:
New factory method to get a validator for 3rd party JWKS clients.
Will add a new keyset column to AuthService to allow selection based on presence.